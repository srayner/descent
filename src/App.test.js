import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Label from "./components/ui/label";

describe("Label component", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Label>My Label</Label>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
