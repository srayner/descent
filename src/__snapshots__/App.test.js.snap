// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component matches the snapshot 1`] = `
<label
  className="sc-bdVaJa bSfUnM"
>
  My Label
</label>
`;
